src/components/Records/tests/Pagination.spec.js

```diff
@@ -114,5 +105,25 @@ describe("Pagination.vue", () => {
     });
 
 
+    it('Has a paginate() method that sets the current page in the URL query', () => {
+
+        $route.query = {page: "12"};
+        $route.name = "Standards";
+        anotherWrapper2 = shallowMount(Pagination, {
+            mocks: {$route, $router},
+            propsData: {
+                totalPages: 10,
+                default: 0
+            }
+        });
+
+        wrapper.vm.$route.query.page = "12";
+        wrapper.vm.currentPage = 2;
+        wrapper.vm.paginate(12);
```

Comment:
paginate() is called but due to throttle the permission method will prevent the code from executing.
This is why paginate will not set the correct wrapper.vm.currentPage

Same for you last() method test. The code inside paginate() is only called for the first() method test. Then, paginate() in all subsequent tests will be prevented by throttle.